modin/pandas/groupby.py

```diff
@@ -294,6 +296,34 @@ def first(self, **kwargs):
     def backfill(self, limit=None):
         return self.bfill(limit)
 
+    _internal_by_cache = no_default
+
+    # TODO: since python 3.9:
+    # @cached_property
+    @property
+    def _internal_by(self):
```

Comment:
I wonder if a more descriptive name could be made...

Reply:
We already have the "internal_by" term integrated into the groupby flow ([[1]](https://github.com/modin-project/modin/blob/b17db22b04b8d2b9133b64a96a7f62f974d9c959/modin/backends/pandas/query_compiler.py#L2660), [[2]](https://github.com/modin-project/modin/blob/b17db22b04b8d2b9133b64a96a7f62f974d9c959/modin/pandas/dataframe.py#L475-L477), [[3]](https://github.com/modin-project/modin/blob/b17db22b04b8d2b9133b64a96a7f62f974d9c959/modin/backends/pandas/query_compiler.py#L2704-L2707)). Do we need a new term describing the same?